larky/src/main/resources/vgs/nts.star

```diff
@@ -72,11 +73,20 @@ def render(
     :param cryptogram_value: JSONPath to insert the cryptogram value of the network token within the input
            payload
     :param cryptogram_eci: JSONPath to insert the cryptogram ECI of the network token within the input payload
+    :param merchant_id: Merchant id to get a network token for
     :return: JSON payload injected with network token values
     """
     pan_value = jsonpath_ng.parse(pan).find(input).value
     amount_value = str(jsonpath_ng.parse(amount).find(input).value)
     currency_code_value = jsonpath_ng.parse(currency_code).find(input).value
+    
+    merchant_id_value = ""
+    merchant_id_result = None
+    if merchant_id != None:
+        merchant_id_result = jsonpath_ng.parse(merchant_id).find(input, error_safe=True)

```

Comment:
it seems a bit odd for me that we are trying to get the get the `merchant_id` value from the payload with `jsonpath`. Usually `render` method will be used with a PSP request payload as the input, and the `merchant_id` is only meaningful to VGS. Now it seems like the customer will need to add unrelative `merchant_id` value in the payload, like

```json
{
    "adyen-payload-goes-here": "...",
    "merchant_id": "MERCHANT_ID"
}
```

and then because the `merchant_id` is not really meant for adyen, we need to pop it out before sending to adyen.

All in all, it seems like forcing `merchant_id` to be read as a jsonpath value is really odd from real world usage perspective. I envision `merchant_id` value should come from header, as it's much easier to deal with, like `VGS-Merchant-Id`, and the code could look like:

```python
    output = nts.render(
        json.loads(input.body),
        pan="$.paymentMethod.number",
        cvv="$.paymentMethod.cvv",
        amount="$.amount.value",
        currency_code="$.amount.currency",
        exp_month="$.paymentMethod.expiryMonth",
        exp_year="$.paymentMethod.expiryYear",
        cryptogram_value="$.mpiData.cavv",
        cryptogram_eci="$.mpiData.eci",
        merchant_id=input.headers.pop("VGS-Merchant-Id", ""),
    )
```